code/controllers/subsystem/atoms.dm

```diff
@@ -28,25 +28,19 @@ SUBSYSTEM_DEF(atoms)
 
 	LAZYINITLIST(late_loaders)
 
-	var/count

```

Comment:
dont change this stuff, especially in a pr like this

Reply:
`pass` was removed. This is dead code.

Reply:
why are you removing it though?

Reply:
like this stuff exists for a reason, why are you trying to remove it in a jobs pr

Reply:
I'll remove it for atomization, but I haven't yet found out the reason this exists beyond being a debugging leftover.
Do you know of a better reason?